gensrc/java/com/starrocks/udf/CallStubGenerator.java

```diff
@@ -0,0 +1,142 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+package com.starrocks.udf;
+
+import jdk.internal.org.objectweb.asm.ClassWriter;
+import jdk.internal.org.objectweb.asm.Label;
+import jdk.internal.org.objectweb.asm.MethodVisitor;
+import jdk.internal.org.objectweb.asm.Type;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+
+import static jdk.internal.org.objectweb.asm.Opcodes.*;
+
+public class CallStubGenerator {
```

Comment:
Can this code generator be simplified?

Reply:
For example, use template based method? Or code-string based method? Instead of generate the byte code?